src/views/Home/index.view.tsx

```diff
@@ -17,7 +28,11 @@ const description = [
 const HomepageView: React.FC = () => {
   return (
     <div className="Homepage">
-      <HeroView pageName={"Home"} title={title} description={description} />
+      <HeroView pageName={"Home"} title={title} description={description}>
+        <div className="Homepage__emailSubscriptionContainer">
+          <EmailSubscriptionForm />
+        </div>
+      </HeroView>
```

Comment:
I decided to move `EmailSubscriptionForm` into `Home` directory/view since it's a Homepage specific component. This will allow us to reuse the `Hero` component when we start to incorporate the different designs for each different page!

Reply:
https://github.com/CruzHacks/cruzhacks-2021-website/blob/3a54fce9690322ac8cea3c319a3c21280246bbd0/src/components/PostcardStack/index.view.tsx#L4-L44

This is an example how the Postcard Stack Component has also been refactored in preparation for the new views! Really dope things we'll be able to do when implementing the rest of the web app.